src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -24,85 +24,70 @@ redirects:
   - /docs/alerts-applied-intelligence
 ---
 
-<LandingPageHero>
-  <HeroContent>
-    Together, New Relic Alerts and Applied Intelligence form a flexible and centralized notification system that unlocks the operational potential of New Relic. Alerts is a single tool to manage alert policies and alert conditions, letting you focus on the metrics you care about most.
+Alerts and applied intelligence are a suite of tools and features that proactively notify you when things go wrong–even when they're starting to go wrong.
 
-    Applied Intelligence helps you find, troubleshoot, and resolve problems faster. Specifically, it's a hybrid machine learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis.
+Alerts lets you define the conditions in your system you want to be notified about, letting you focus on the things you care about most.
 
-    <ButtonGroup>
+Applied intelligence is a machine-learning engine that reduces alert noise, correlates incidents, automatically detects anomalies, and provides root cause analysis. Not only that, it enriches your notifications with metadata to give you greater context of what's happening when you are notified.
+
+<ButtonGroup>
     <ButtonLink
     role="button"
     to="https://newrelic.com/signup"
     variant="primary"
     >
     Sign up for free
     </ButtonLink>
-    </ButtonGroup>
+</ButtonGroup>
     No credit card required. Already have an account? [Login](http://one.newrelic.com/).
 
-  </HeroContent>
-
 ![Screenshot of the Alerts and AI overview page](./images/alerts-ai-summary.png 'alerts-ai.png')
 
   <figcaption>
     **[one.newrelic.com](https://one.newrelic.com) > Alerts & AI: You can use the overview summary page to quickly review your violations, issues, and incidents.**
```

Comment:
Should we be talking about both violations and incidents? In a post-photon world, they are the same thing, but referring to the same thing as both "violations" and "incidents" may cause confusion.

Reply:
I shudder to think of how many times we mention violations in our alerts docs. Something to put on the old To Do list.